docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -49,6 +53,12 @@ deactivate AddressBookParser
 LogicManager -> DeleteCommand : execute(m)
 activate DeleteCommand
 
+DeleteCommand -> Ui : showAlertDialogAndWait()
+activate Ui
+
+Ui --> DeleteCommand
+deactivate Ui
```

Comment:
Also, is there a way to remove the lifeline of activation bar once you have deleted?

Reply:
In the developer guide, it is stated that the lifeline is a limitation of PlantUML so it will not be considered a bug.